### PR TITLE
Paulmoloneyr3/doc 4642/port topics to enterprise

### PR DESCRIPTION
**Pull request by paulmoloneyr3**
Merged missing topics previously in Community/Open Source into Enterprise. Fixed resulting links and menus.

### CODE REVIEW

File: content/en/platform/corda/4.10/enterprise/app-upgrade-notes-enterprise.md [hunk 1]
```diff
@@ -81,7 +81,7 @@ You can do either of the following:
 * Upgrade your `quasar.jar` file to the version consistent with your Corda version.
 * Delete your `lib` directory and switch to using the Gradle test runner.
 
-You can find instructions for both options in [Running tests in IntelliJ](../community/testing.html#running-tests-in-intellij).
+You can find instructions for both options in [Running tests in IntelliJ](testing.html#running-tests-in-intellij).
```
Comment (nadinequinr3) [inline content/en/platform/corda/4.10/enterprise/app-upgrade-notes-enterprise.md:84]:
```suggestion
   You can find instructions for both options in [Running tests in IntelliJ](testing.html#running-tests-in-intellij).
```

Comment (paulmoloneyr3) [reply content/en/platform/corda/4.10/enterprise/app-upgrade-notes-enterprise.md:84]: Done.

**Comment by nadinequinr3** [comment]
I'm not sure how far we want to go at this stage but couple of thoughts from looking at the preview:
* The Key Concepts section seems a bit buried to me. Would it be more appropriate in About Corda maybe?
* The first "Introduction to **Corda"**" page seems a bit pointless. Should that be merged with "About Corda"?

**Comment by paulmoloneyr3** [comment]
> 

I'll set those up as separate tickets to look at.

**Comment by colin-hillis** [comment]
> I'm not sure how far we want to go at this stage but couple of thoughts from looking at the preview:
> 
> * The Key Concepts section seems a bit buried to me. Would it be more appropriate in About Corda maybe?
> * The first "Introduction to Corda" page seems a bit pointless. Should that be merged with "About Corda"?

@nadinequinr3 and @paulmoloneyr3 I definitely agree with both in principle, but some of these restructuring tickets are getting pretty complex. Restructuring like this is a very deep rabbit hole that we should tackle in chunks. The more you do, the more you find. There is so much extra that I saw that could be done in my restructure piece alone, like the pointless "Introduction to Corda" page (which is not as easy to remove as first thought), but you have to pull back and focus on the main bits. Other smaller structural tweaks, deletions and re-writes of poorly thought out content can come later.

**Comment by paulmoloneyr3** [comment]
> > I'm not sure how far we want to go at this stage but couple of thoughts from looking at the preview:
> > 
> > * The Key Concepts section seems a bit buried to me. Would it be more appropriate in About Corda maybe?
> > * The first "Introduction to Corda" page seems a bit pointless. Should that be merged with "About Corda"?
> 
> @nadinequinr3 and @paulmoloneyr3 I definitely agree with both in principle, but some of these restructuring tickets are getting pretty complex. Restructuring like this is a very deep rabbit hole that we should tackle in chunks. The more you do, the more you find. There is so much extra that I saw that could be done in my restructure piece alone, like the pointless "Introduction to Corda" page (which is not as easy to remove as first thought), but you have to pull back and focus on the main bits. Other smaller structural tweaks, deletions and re-writes of poorly thought out content can come later.

I fixed the Key Concepts issue as that was an easy one. Can add the other issue as a separate task.

**Review by nadinequinr3 — APPROVED** [review]

**Review by yarivus — APPROVED** [review]
LGTM

File: content/en/archived-docs/corda-os/4.7/key-concepts-transactions.md [hunk 2]
```diff
@@ -3,11 +3,11 @@ aliases:
 - /head/key-concepts-transactions.html
 - /HEAD/key-concepts-transactions.html
 - /key-concepts-transactions.html
-date: '2020-04-07T12:00:00Z'
+date: '2023-02-01'
 menu:
-  corda-os-4-7:
-    identifier: corda-os-4-7-key-concepts-transactions
-    parent: corda-os-4-7-key-concepts
+  corda-enterprise-4-7:
```
[the remark this answers is not in the document]
Comment (paulmoloneyr3) [reply content/en/archived-docs/corda-os/4.7/key-concepts-transactions.md:8]: You're right, accidentally edit there. I'll fix that post-merge and also review the other docs there.

[the remark this answers is not in the document]
Comment (paulmoloneyr3) [reply content/en/archived-docs/corda-os/4.7/key-concepts-transactions.md:8]: https://github.com/corda/corda-docs-portal/pull/1011/files

**Pull request merged**